lib/releaselib.sh

```diff
@@ -480,23 +480,20 @@ release::gcs::copy_release_artifacts() {
 
   # Copy the main set from staging to destination
   logecho -n "- Copying release artifacts to $gcs_destination: "
-  logrun -s $GSUTIL -qm cp -r $gcs_stage/* $gcs_destination/ || return 1
+  logrun -s $GSUTIL -qm cp -a public_read -r $gcs_stage/* $gcs_destination/ || return 1
 
-  logecho -n "- Marking all uploaded objects public: "
-  logrun -s $LOGRUN_MOCK $GSUTIL -q -m acl ch -R -g all:R \
-                                 "$gcs_destination" || return 1
+  # This small sleep gives the eventually consistent GCS bucket listing a chance
+  # to stabilize.
+  sleep 5
```

Comment:
Neat, I didn't find any mentions of "Mirroring build" in PR or CI jobs.

Should I leave the sleep in, accept that the "listing final contents" step might be flaky, or remove it?